tests/apps/service_invocation_grpc_proxy_server/service.yaml

```diff
@@ -20,7 +20,6 @@ spec:
         dapr.io/enabled: "true"
         dapr.io/app-id: "grpcproxyserver"
         dapr.io/app-port: "50051"
-        dapr.io/app-protocol: "grpc"
```

Comment:
Why reverting this? Is it a left over from debugging?

Reply:
Redundant, was never needed.